docs/docs/markers.mdx

```diff
@@ -0,0 +1,321 @@
+
+The negated forms of the labels are also supported:
+
+- `not_action`: the event is not the specified bot action.
+- `not_intent`: the event is not the specified user intent.
+- `slot_was_not_set`: the specified slot has not been set.
+
+### Operators
+
+The following operators are supported:
+
+- `and`: all listed conditions applied.
+- `or`: any of the listed conditions applied.
+- `not`: the condition did not apply. This operator only accepts 1 condition.
+- `seq`: the list of conditions applied in the specified order, with any number of events occurring in-between.
+- `at_least_once`: the listed marker definitions occurred at least once. Only the first occurrence will be marked.
+- `never`: the listed marker definitions never occurred.
+
+### Marker Configuration
+
+Here is an example of a marker configuration file containing several marker definitions. The example is created for mood bot,
+with a new slot `name` to illustrate the use of the label `slot_was_set`:
+
+```yaml
+marker_name_provided:
+  slot_was_set: name
+
+marker_mood_expressed:
+  or:
+    - intent: mood_unhappy
+    - intent: mood_great
+
+marker_cheer_up_failed:
+  seq:
+    - intent: mood_unhappy
+    - action: utter_cheer_up
+    - action: utter_did_that_help
+    - intent: deny
+
+marker_bot_not_challenged:
+  never:
+    - intent: bot_challenge
+
+marker_cheer_up_attempted:
+  at_least_once:
+    - action: utter_cheer_up
+
+marker_mood_expressed_and_name_not_provided:
+  and:
+    - not:
+      - slot_was_set: name
+    - or:
+      - intent: mood_unhappy
+      - intent: mood_great
+```
+Note the following:
+
+- Each marker has a unique identifier (or name) such as `marker_name_provided`.
+
+- A marker definition can contain a single condition, as shown in `marker_name_provided`.
+
+- A marker definition can contain a single operator with a list of conditions, as shown in `marker_mood_expressed`,
+`marker_cheer_up_failed`, `marker_bot_not_challenged`, and `marker_cheer_up_attempted`.
+
+- A marker definition can contain nested operators, as shown in `marker_mood_expressed_and_name_not_provided`.
+
+- The values assigned to event conditions must be valid according to your bot's `domain.yml` file. For example, in
+`marker_mood_expressed`, the intents `mood_unhappy` and `mood_unhappy` are both intents listed in the mood bot's `domain.yml` file.
+
+:::note
+You cannot yet reuse an existing marker name in the definition of another marker.
+:::
+
+## Extracting Markers
+
+Markers are extracted from dialogues already stored in a tracker store. To learn how to store interactions with your bot in a tracker store,
+read the [Tracker Store](./tracker-stores.mdx) page.
+
+Once you've created your marker definitions in the marker configuration file, and have stored some dialogues in your tracker store,
+you can apply your markers to your trackers by running the following command:
+
+```bash
+rasa evaluate markers all --config marker.yml extracted_markers.csv
```

Comment:
The CLI section suggests that we assume `markers.yml` as a default value. Would be best to stick to that filename.

Reply:
```suggestion
rasa evaluate markers all --config markers.yml extracted_markers.csv

```